src/Tribe/Views/V2/View.php

```diff
@@ -787,6 +787,9 @@ public function next_url( $canonical = false, array $passthru_vars = [] ) {
 				$url = add_query_arg( $query_args, $url );
 			}
 
+			// Remove the inverse of the page key we are using.
+			$url = remove_query_arg( 'page' === $this->page_key ? 'paged' : 'page', $url );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Usage of Yoda conditions is not allowed; switch the expression order
* :warning: Line exceeds 80 characters; contains 91 characters

<sub>hash: b0b4ec7b93cc1d38d1075a3c5a24871b</sub>